ertviz/controllers/multi_parameter_controller.py

```diff
@@ -65,19 +58,25 @@ def _update_histogram(parameter, hist_check_values, selected_ensembles):
             raise PreventUpdate
         data = {}
         colors = {}
+        priors = {}
         for ensemble_id, color in selected_ensembles.items():
-            if parameter in parent.parameter_models[ensemble_id]:
-                data[parent.ensembles[ensemble_id]._name] = parent.parameter_models[
-                    ensemble_id
-                ][parameter].data_df()
-                colors[parent.ensembles[ensemble_id]._name] = color["color"]
+            ensemble = load_ensemble(parent, ensemble_id)
+            if parameter in ensemble.parameters:
+                parameter_model = ensemble.parameters[parameter]
+                data[ensemble._name] = parameter_model.data_df()
+                colors[ensemble._name] = color["color"]
+
+                if parameter_model.priors and "prior" in hist_check_values:
+                    priors[ensemble._name] = (parameter_model.priors, color)
```

Comment:
Hm, do you think that prior can change over different ensembles for the same parameter?